src/Exceptions/InvalidArgumentObjectsException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Exceptions;
+
+final class InvalidArgumentObjectsException extends AlgoliaException
+{
+    public function __construct($message = '', $code = 0, $previous = null)
+    {
+        if (!$message) {
```

Comment:
This is type juggling. Please use:

```suggestion
        if ($message === '') {
```